bgs/handlers.go

```diff
@@ -6,36 +6,33 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+	"strconv"
 	"strings"
 
 	atproto "github.com/bluesky-social/indigo/api/atproto"
 	comatprototypes "github.com/bluesky-social/indigo/api/atproto"
+	"gorm.io/gorm"
+
 	"github.com/bluesky-social/indigo/util"
 	"github.com/bluesky-social/indigo/xrpc"
+	"github.com/ipfs/go-cid"
 	"github.com/labstack/echo/v4"
 )
 
+// Since the removal of repo history, this is the same as GetRepo but without the "since" bit
```

Comment:
should just remove this endpoint

Reply:
Yeah I think so, it'll be deprecated soon